installer/dbscript/dbscript-2.sql

```diff
@@ -3619,7 +3618,8 @@ VALUES ('apiv2_admin_education', 'API-v2 Admin - Education', 1, 1, 1, 1),
        ('apiv2_admin_paygrade_currency', 'API-v2 Admin - Pay Grade Currencies', 1, 1, 1, 1),
        ('apiv2_admin_skill', 'API-v2 Admin - Skills', 1, 1, 1, 1),
        ('apiv2_admin_subunit', 'API-v2 Admin - Organization Structure', 1, 1, 1, 1),
-       ('apiv2_admin_user', 'API-v2 Admin - Users', 1, 1, 1, 1);
+       ('apiv2_admin_user', 'API-v2 Admin - Users', 1, 1, 1, 1),
+       ('apiv2_email_configuration', 'API-v2 Admin - Email Configuration', 1, 0, 1, 0);
```

Comment:
can you refactor as `apiv2_admin_email_configuration` so it will reflect the module name